src/lib/Component/Content/ContentEditMetaFieldsComponent.php

```diff
@@ -70,4 +77,21 @@ private function getMetaFieldTypeIdentifiers(): array
             )
         );
     }
+
+    private function getMetaFieldDefinitionCollection(ContentType $contentType): FieldDefinitionCollection
+    {
+        $metaFieldGroups = $this->configResolver->getParameter(
+            'admin_ui_forms.content_edit.meta_field_groups_list'
+        );
+
+        return $contentType->fieldDefinitions->filter(
+            static fn (FieldDefinition $field): bool => true === in_array($field->fieldGroup, $metaFieldGroups),
```

Comment:
`in_array` already returns a boolean, so comparison is unnecessary.

Using 3rd parameter for `in_array` function (loose/strict comparison) is recommended.

```suggestion
            static fn (FieldDefinition $field): bool => in_array($field->fieldGroup, $metaFieldGroups, true),
```